components/prf/hmac-sha256/src/prf.rs

```diff
@@ -152,6 +178,10 @@ where
             .await
     }
 
+    #[cfg_attr(
+        feature = "tracing",
+        tracing::instrument(level = "debug", skip(self, handshake_hash), err)
```

Comment:
for consistency, let's **NOT** skip `handshake_hash` here since we are not skipping it elsewhere